zoneinfo/src/lib.rs

```diff
@@ -0,0 +1,505 @@
+// Implementation note: this library is NOT designed to be the most
```

Comment:
suggestion: It would be helpful to amend this comment or to write a doc comment for the module as a whole about why this module is in use over `parse-zoneinfo`. It would serve as a nice architectural record and maybe stop folks like me from asking silly questions.